### DISCUSSION TITLE
Getting error while pulling in [@types/geojson] package.

### DISCUSSION THREAD

**Discussion by snehasis** [DefinitelyTyped/DefinitelyTyped#61226] [Issues with a @types package]
Getting below error which is a dependancy of @types/highcharts@4.2.57

npm ERR! path c:\jenkins\workspace\node_modules\.staging\@types\geojson-fce33bf7\package.json
npm ERR! code ENOTDIR
npm ERR! errno -4058
npm ERR! syscall open

npm ERR! ENOENT: no such file or directory, open 'c:\jenkins\workspace\node_modules\.staging\@types\geojson-fce33bf7\package.json'

[a comment by an automated account is not shown]

**Reply by JeffJacobson** [reply]
> Getting below error which is a dependancy [sic] of @types/highcharts@4.2.57
> 
> npm ERR! path c:\jenkins\workspace\node_modules.staging@types\geojson-fce33bf7\package.json npm ERR! code ENOTDIR npm ERR! errno -4058 npm ERR! syscall open
> 
> npm ERR! ENOENT: no such file or directory, open 'c:\jenkins\workspace\node_modules.staging@types\geojson-fce33bf7\package.json'

This looks like an issue with [Jenkins](https://www.jenkins.io/) or its configuration rather than the GeoJSON type definition.

**Reply by icholy** [reply]
Ya I can't reproduce locally. I'm not sure where that `-fce33bf7` suffix is coming from.
